.travis.yml

```diff
@@ -34,7 +34,7 @@ jobs:
     - stage: checkstyle
       script: mvn checkstyle:checkstyle
     - stage: copyright
-      script: mvn glassfish-copyright:check
+      script: bash copyright.sh
```

Comment:
can we move the copyright and checkstyle checks to run before the `install` check? This is because they will commonly fail when PRs are first opened and they run fast, so it will help our travis builds "fail fast" if there is a problem with the checks

Reply:
Sure :-)